### PR TITLE
A few n/m/block and n/m/inventory mappings

### PR DESCRIPTION
**Pull request by IotaBread**
What was named `JukeboxInventory` was actually used in `HorseBaseEntity`, and its subclass in `JukeboxBlockEntity` and `DecoratedPotBlockEntity`

### CODE REVIEW

File: mappings/net/minecraft/inventory/Inventory.mapping
Comment (IotaBread) [inline mappings/net/minecraft/inventory/Inventory.mapping:1]: This conflicted with #540 so I added the relevant changes here

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/block/TrialSpawnerBlock.mapping [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/unmapped/C_qiypfnai net/minecraft/block/TrialSpawnerBlock
+	FIELD f_eqxijsod STATE Lnet/minecraft/unmapped/C_cgckxfsw;
+	METHOD m_hqexpyll (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_ndgqjckh;)V
+		ARG 1 world2
```
Comment (eristhea) [inline mappings/net/minecraft/block/TrialSpawnerBlock.mapping]: world2?

**Review by eristhea — COMMENTED** [review]
slay

File: mappings/net/minecraft/block/TrialSpawnerBlock.mapping (hunk 1 above)
Comment (IotaBread) [reply mappings/net/minecraft/block/TrialSpawnerBlock.mapping]: It's a lambda, `world` exists in the outer method, and I didn't know what to name this one (it isn't used anyway)

Comment (supersaiyansubtlety) [reply mappings/net/minecraft/block/TrialSpawnerBlock.mapping]: How about unused?

Comment (IotaBread) [reply mappings/net/minecraft/block/TrialSpawnerBlock.mapping]: `unusedWorld` or just `unused`?

Comment (eristhea) [reply mappings/net/minecraft/block/TrialSpawnerBlock.mapping]: `unusedWorld` imo

Comment (IotaBread) [reply mappings/net/minecraft/block/TrialSpawnerBlock.mapping]: Fixed, btw

**Review by eristhea — APPROVED** [review]

**Pull request merged**